erpnext/hr/doctype/leave_policy/leave_policy_dashboard.py

```diff
@@ -8,13 +8,17 @@ def get_data():
 		},
 		'transactions': [
 			{
-				'items': ['Employee']
-			},
-			{
-				'items': ['Employee Grade']
+				'label': ('Employees'),
```

Comment:
```suggestion
				'label': _('Employees'),
```
Make labels translatable